packages/inventory-compliance/src/SystemPolicyCard.js

```diff
@@ -39,31 +40,31 @@ class SystemPolicyCard extends React.Component {
         return (
             <Card>
                 <CardBody>
-                    <TextContent>
-                        <Text style={ { marginBottom: '0' } } component={ TextVariants.small }>External Policy</Text>
-                        <Text style={ { marginTop: '0' } } component={ TextVariants.h4 }>{ this.state.policy.name }</Text>
+                    <TextContent style={ { marginBottom: 'var(--pf-global--spacer--md)' } } >
+                        <Text style={ { marginBottom: '0' } } component={ TextVariants.small }>External policy</Text>
+                        <Text style={ { marginTop: '0', marginBottom: '0' } } component={ TextVariants.h4 }>{ this.state.policy.name }</Text>
                     </TextContent>
-                    { this.complianceIcon(this.state.policy.compliant) }
-                    <Text component={ TextVariants.small }>
-                        { this.state.policy.rules_passed } of { this.state.policy.rules_passed + this.state.policy.rules_failed } rules passed
-                    </Text>
-                    <Text
-                        component={ TextVariants.medium }
-                        onMouseEnter={ this.onMouseover.bind(this) }
-                        onMouseLeave={ this.onMouseout.bind(this) }
-                        style={ { wordWrap: 'break-word' } }
-                    >
-                        Profile <br/>
-                        { this.state.refIdTruncated }
-                    </Text>
-                </CardBody>
-                <CardFooter>
-                    <TextContent>
+                    <div style={ { marginBottom: 'var(--pf-global--spacer--md)' } } >
+                        { this.complianceIcon(this.state.policy.compliant) }
                         <Text component={ TextVariants.small }>
-                          Last scanned: { this.state.policy.last_scanned }
+                            { this.state.policy.rules_passed } of { this.state.policy.rules_passed + this.state.policy.rules_failed } rules passed
                         </Text>
-                    </TextContent>
-                </CardFooter>
+                    </div>
+                    <div style={ { marginBottom: 'var(--pf-global--spacer--md)' } } >
+                        <Text
+                            component={ TextVariants.medium }
+                            onMouseEnter={ this.onMouseover.bind(this) }
```

Comment:
You used fat arrow previous, no need to bind this.

Reply:
Hmm... it doesn't seem to work without these two binds (onMouseEnter/Leave). Any idea?

Reply:
So changing it to `onMouseEnter={ this.onMouseover }` or `onMouseEnter={() => this.onMouseover() }` is not helping?

Reply:
Oh, I removed the two lines completely :laughing: Yes, it works with `onMouseEnter={ this.onMouseover }`. Of course, I didn't change any of these lines in this PR, so it's kind of out of scope. But, I've added it as a separate commit.